pkg/api/bytes.go

```diff
@@ -66,6 +67,16 @@ func (s *server) bytesUploadHandler(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+
+	if strings.ToLower(r.Header.Get(SwarmPinHeader)) == "true" {
```

Comment:
we need to add this logic also to:
`bzz POST` handler - this should be in both forks - file upload and dir upload handler
`chunk POST` handler
`feed POST` handler
`soc POST` handler